NitroxServer/Communication/Packets/Processors/PvPAttackProcessor.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections.Generic;
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+using NitroxServer.Serialization;
+
+namespace NitroxServer.Communication.Packets.Processors;
+
+public class PvPAttackProcessor : AuthenticatedPacketProcessor<PvPAttack>
+{
+    private readonly ServerConfig serverConfig;
+    private readonly PlayerManager playerManager;
+
+    // TODO: In the future, do a whole config for damage sources
+    private static readonly Dictionary<PvPAttack.AttackType, float> damageMultiplierByType = new()
+    {
+        { PvPAttack.AttackType.KnifeHit, 0.5f },
```

Comment:
Why do we need a multiplier and not just use the base value of the weapon?

Reply:
I think it makes sense to reduce the damage of starter weapon so new players can ease into Pvp accidents.

Reply:
Yeah makes sense. I think having that config Todo created then for future would be good for players to configure but for now this is a good start to the pvp sync